crypto/core_namemap.c

```diff
@@ -335,3 +303,107 @@ int ossl_namemap_add_names(OSSL_NAMEMAP *namemap, int number,
 
     return number;
 }
+
+/*-
```

Comment:
`/*-` to open a multi-line comment?  I thought it was just `/*`

Reply:
The dash is an accepted character to tell the 'indent' program not to reformat the comment.

Reply:
Cool.